service/controller/releasecycle/resource/app/create.go

```diff
@@ -2,62 +2,61 @@ package app
 
 import (
 	"context"
-	"fmt"
 
 	applicationv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
 	"github.com/giantswarm/microerror"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/giantswarm/release-operator/service/controller/key"
 )
 
+// ApplyCreateChange ensures createChange App CR is created in k8s api.
```

Comment:
```suggestion
// ApplyCreateChange ensures the App CR is created in the k8s api.
```

I'd actually be fine without the godoc here as this is standard operatorkit behaviour. I usually just add godoc for the resource constructor.